python/cudf/cudf/tests/test_struct.py

```diff
@@ -392,3 +392,13 @@ def test_struct_with_null_memory_usage():
 
     s[2:4] = None
     assert s.memory_usage() == 272
+
+
+@pytest.mark.parametrize("zlice", [slice(0, 3), slice(1, 4)])
```

Comment:
Can we use `slice_` instead of… _zlice_ ?

Reply:
Done.